eventhubs-streamanalytics-eventhubs/tools/README.md

```diff
@@ -1,11 +1,46 @@
-# Testing End-To-End Latency
+---
+page_type: sample
+languages:
+- csharp
+products:
+- azure
+- azure eventhubs 
+extensions:
+- platforms: dotnet
+---
 
-Get the Event Hub connection string created and displayed during the script execution. If you cannot find it, just grab the execution string for the `Listen` Shared Access Policy for the output eventhub.
+# Getting started on processing received message from Event Hubs #
 
+Azure eventhubs-streamanalytics-eventhubs sample for processing received message from Event Hubs -
+ - Create an EventHubConsumerClient based on Event Hub-level connection string.
+ - Read events from EventHubConsumerClient.
+ - Processing received message and write to a .csv file.
+
+# Running this Sample #
+To run this sample:
```

Comment:
I don't see anything here that discusses ASA and setting up ASA to publish to the Event Hubs namespace.  Without that, this sample really doesn't do anything since it doesn't publish any events for itself to read.

Reply:
Yes, without publishing to the Event Hubs namespace, this sample does not do anything,
this is same with the origin logic(T1 pattern). When i am handling this in new logic(T2 pattern), a sender simulation was being created by myself. Since simulation is used to make sure all logic in T2 pattern is working as expected and did not mention in origin T1 pattern also, i decide to not disscuss about publishing to the Event Hubs namespace.

Do you agree with this?

Reply:
I'm good with not elaborating on detail, but it's probably worth a quick mention that:  "You should have an existing Azure Stream Analytics configuration that publishes to this event hub."  or something similar.